From 6c1c90b61f9ab8fe04077f068a15ff7b27b31ded Mon Sep 17 00:00:00 2001
From: "Li, Ning" <ning.li@intel.com>
Date: Sun, 3 Nov 2013 12:22:43 +0800
Subject: [PATCH 149/429] flis: Move the FLIS driver initialization to an
 earlier phase

On MRFL, FLIS driver API is called by GPIO driver.
We have to move flis driver initialization earlier.

Signed-off-by: Li, Ning <ning.li@intel.com>
---
 drivers/platform/x86/intel_scu_flis.c | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/drivers/platform/x86/intel_scu_flis.c b/drivers/platform/x86/intel_scu_flis.c
index a1b120ca3fd6..4dc3598d72c9 100644
--- a/drivers/platform/x86/intel_scu_flis.c
+++ b/drivers/platform/x86/intel_scu_flis.c
@@ -739,11 +739,7 @@ static void __exit flis_rpmsg_exit(void)
 	return unregister_rpmsg_driver(&flis_rpmsg);
 }
 
-#ifdef MODULE
-module_init(flis_rpmsg_init);
-#else
-rootfs_initcall(flis_rpmsg_init);
-#endif
+fs_initcall(flis_rpmsg_init);
 module_exit(flis_rpmsg_exit);
 
 MODULE_AUTHOR("Ning Li <ning.li@intel.com>");
-- 
2.37.3

